test/library/packages/UnitTest/UnitTest.chpl

```diff
@@ -729,6 +774,11 @@ module UnitTest {
       PrintError(reason);
     }
 
+    proc addIncorrectNumLocales(test, reason) throws {
+      stdout.writeln("IncorrectNumLocales");
+      PrintError(reason);
```

Comment:
This doesn't use the test argument, is that intentional?

Reply:
Yes, I kept for case we may need to use the test name later.